recipes.mdwn

```diff
@@ -22,6 +22,7 @@ to improve your Awesome setup.
 * [Battery Indicator (UPower)](https://github.com/stefano-m/awesome-power_widget)
 * [[Google Play Music Desktop Player|recipes/gpmdp]]
 * [Set of simple widgets](https://github.com/streetturtle/awesome-wm-widgets) - widgets for battery, cpu, brightness, volume, email, etc.
+* [Wireless status](../recipes/wirelessStatus)
```

Comment:
Can't you link to it in the same way that the `Google Play Music Desktop Player` works above? You seem to use the syntax for external links.